dojo/fixtures/test_type.json

```diff
@@ -563,6 +563,13 @@
         },
         "model": "dojo.test_type",
         "pk": 172
-    }
+		},
+	{
+			"fields": {
+					"name": "Gitleaks Scan"
+			},
+			"model": "dojo.test_type",
+			"pk": 173
+	}
 ]
```

Comment:
indentation seems to be off

Reply:
I fixed the indentation in the new commit. Thanks.